framework/resources/Functional/query_parser/DRILL-6974_alter_session_set_command_works.sql

```diff
@@ -0,0 +1,3 @@
+set  store.kafka.poll.timeout = 201;
+alter session set store.kafka.poll.timeout;
```

Comment:
Without value, is the set command valid? I haven't looked at recent changes

Reply:
SET "PROPERTY_NAME" ;   (without value) is new functionality that was implemented in scope of DRILL-6974. It returns current value of property that is set at current time.

Reply:
Thanks for the clarification!